src/pages/mesh/advanced/caching/index.md

```diff
@@ -75,11 +94,27 @@ When the response includes cache-control values, only the [most restrictive valu
 
 <InlineAlert variant="info" slots="text"/>
 
-POST requests are not supported, and GET requests are limited to 2,048 characters.
+GET requests are limited to 2,048 characters.
```

Comment:
This can be moved above after the `Enable caching` example.

Also, the `x-include-metadata` inline alert above relates to response headers. I think we can move it under `How conflicting header values are resolved` as a way to inspect the cache-control headers coming back from each source.